lib/commands/WeatherCommand.js

```diff
@@ -0,0 +1,27 @@
+const weather = require('openweathermap-js')
+
+const Command = require('./Command')
+
+module.exports = new Command({
+	name: 'weather',
+	command: getWeather
+})
+
+function getWeather(context, zip) {
+	const { logger, message } = context 
+	weather.defaults({
+		appid: '6382f16f08da7243484a4041a62ce7da',
+		ZIPcode: '02532,us',
+		method: 'zip'
+	})
+
+	weather.current({ method: 'zip', ZIPcode: zip, units: 'imperial' }, function(err, data) {
+		if (err) {
+			logger.error(err.toString())
+		} else {
+			console.log(data)
```

Comment:
Can you remove this log line? We will already log out the relevant data when we send the message.